main.go

```diff
@@ -112,6 +112,7 @@ func init() {
 
 	flag.BoolVar(&config.GatherSessionTicket, "tls-session-ticket", false, "Send support for TLS Session Tickets and output ticket if presented")
 	flag.BoolVar(&config.ExtendedMasterSecret, "tls-extended-master-secret", false, "Offer RFC 7627 Extended Master Secret extension")
+	flag.BoolVar(&config.OutputCryptoVariables, "output-crypto-variables", false, "Explicitly add crypto variables (PMS & MS) to JSON output")
```

Comment:
@dadrian and I previously discussed possibly adding this to the default output but no final decision was made. Is that still a possibility or are we going w/ "--out-master-secret"?